draft-ietf-detnet-dp-sol-mpls-02.xml

```diff
@@ -484,51 +576,107 @@ System  |    +--------+       +--------+       +--------+   | System
              |<------ End to End DetNet Service ------->|
+
+    X   = Optional service protection (none, PRF, PREOF, PEF/POF)
+    DFx = DetNet member flow x over a TE LSP
 ]]></artwork>
 </figure>
- <t>
-  <xref target="fig_ip_pw_detnet"/> illustrates how end to end DetNet
-  service is provided where the end systems are able to send and receive
-  IP DetNet flows, e.g., per <xref target="I-D.ietf-detnet-dp-sol-ip"/>, and
-  the MPLS  nodes optionally provide service protection. In this
-  case R1 and R3 are T-PEs and R2 is an S-PE and the DetNet service is
-  end-to-end.
- </t>
+  </section>
+  <section title="IEEE 802.1 TSN Over MPLS DetNet Data Plane Scenario"
+             anchor="sec_tsn_mpls_dt_dp_scen">
 
-<figure align="center" anchor="fig_ip_pw_detnet"
- title="DetNet IP over DetNet (DN) MPLS">
-<artwork><![CDATA[
-      DetNet                                         DetNet
-IP    Service         Transit          Transit       Service  IP
-DetNet               |<-Tnl->|        |<-Tnl->|               DetNet
-End     |            V   1   V        V   2   V            |  End
-System  |   +--------+       +--------+       +--------+   |  System
-+---+   |   |   R1   |=======|   R2   |=======|   R3   |   |   +---+
-|   |-------|._X_....|..DF1..|.__ ___.|..DF3..|...._X_.|-------|   |
-|CE1|   |   |    \   |       |   X    |       |   /    |   |   |CE2|
-|   |   |   |     \_.|..DF2..|._/ \__.|..DF4..|._/     |   |   |   |
+
+<figure anchor="fig_tsn_mpls_detnet" align="center"
+title="A TSN over DetNet MPLS Enabled Network">
+<artwork align="center"><![CDATA[
+ TSN             Edge          Transit        Edge        TSN
+ End System      Node           Node          Node        End System
+                (T-PE)         (LSR)          (T-PE)
+
++----------+  +.........+                   +.........+  +----------+
+|   Appl.  |<-:Svc Proxy:--End to End Svc.--:Svc Proxy:->|   Appl.  |
++----------+  +---------+                   +---------+  +----------+
+|    TSN   |  |TSN| |Svc|<-- DetNet flow -->: Service :->|    TSN   |
++----------+  +---+ +---+    +----------+   +---------+  +----------+
+|Forwarding|  |Fwd| |Fwd|    |Forwarding|   |Fwd| |Fwd|  |Forwarding|
++------.---+  +--.+ +-.-+    +---.----.-+   +--.+ +-.-+  +----.-----+
+       :   Link  :    /  ,-----.  \   :  Link  :   /  ,-----.  \
+       +.........+    +-[  Sub  ]-+   +........+   +-[  Sub  ]-+
+                        [Network]                    [Network]
+                         `-----'                      `-----'
+
+        |<- TSN ->| |<------ DetNet MPLS ------>| |<-- TSN --->|
+]]></artwork>
+</figure>
+
+  <t>
+   <xref target="fig_tsn_mpls_detnet"/> shows IEEE 802.1 TSN end
+   stations operating over a TSN aware DetNet service running over an
+   MPLS network.  DetNet Edge Nodes sit at the boundary of a DetNet
+   domain. They are responsible for mapping non-DetNet aware traffic to
+   DetNet services.  They also support the imposition and disposition of

```

Comment:
Add Layer-2:
"They are responsible for mapping non-DetNet aware **L2** traffic to"